### PR TITLE
Ron's updates after our hangout

### PR DESCRIPTION
**Pull request by smellular**
- callToAction on index.html
- a:hover text-shadow has been replaced with underline

### CODE REVIEW

**Comment by smellular** [comment]
:point_right:@stevenhauser:point_left:  I know these updates can be done better so I'll wait for your review before merging. Everything we did during our hangout is now live.

Do I even need to tag you in these comments or do you get notified when I commit changes?

**Comment by smellular** [comment]
She doesn't like my changes. She prefers what is live now.

File: beginners-yoga-course.html [hunk 1]
```diff
@@ -36,16 +36,17 @@ <h2 class="headline inverted uppercase program__title">Beginners Yoga Course</h2
...
+      <a class="callToAction-register" target="_blank"
+      href="https://docs.google.com/forms/d/1TUbJxIprhOcw02V6gS4UUgb7r8Kxm6rBtWOteg62ddw/viewform">
+      <span class="unbreakable">Register by April 1st </span>
+      <span class="unbreakable">to save your spot!</span>
+    </a>
+
       <div class="content">
         <p>This course is a step-by-step introduction to Sivananda Classical Hatha Yoga. Students will experience the Five Points of Yoga including proper exercise, breathing, diet, relaxation, and meditation. Sivananda yoga guides students through asanas, while relaxing between each posture. Classes begin with chanting, pranayama breathing, sun salutations and go into the 12 basic asanas. This course walks students through at their own pace and completion of the course will provide a full understanding of the Sivanada Open Class.</p>
-        <p>Course is limited to 10 students per course.</p>
+        <p class="crappy-class-name">Course is limited to 10 students per course.</p>
```
Comment (stevenhauser) [inline beginners-yoga-course.html]: Perfecto.

**Comment by smellular** [comment]
I think this is a good chance for me to learn how to merge specific changes I've made since the last merge. Of all the new changes, I'd only like to keep the a:hover changes. Is there an easy way to do this?

**Comment by stevenhauser** [comment]
There's not a way to do that though the desktop app, no. You'd have to do it through the command line. For now I'd suggest you just copy your changes to a new branch manually.

**Pull request closed without merging**